solution/ui/e2e/cypress/integration/part.spec.js

```diff
@@ -48,7 +49,11 @@ describe("Part View", () => {
 
         cy.focused().then(($el) => {
             cy.get($el).should("have.id", "433-51");
-            cy.get($el).should("have.css", "background-color", "rgb(238, 250, 254)");
+            cy.get($el).should(
+                "have.css",
+                "background-color",
+                "rgb(238, 250, 254)"
+            );
```

Comment:
Nice job making it cleaner and more readable.

Reply:
Prettier made it prettier!